apps/api/src/domain/models/user-management/utilities/coscrad-date.entity.ts

```diff
@@ -140,6 +140,19 @@ export class CoscradDate extends BaseDomainModel {
         return date;
     }
 
+    static fromUnixTimestamp(timestamp: number): ResultOrError<CoscradDate> {
```

Comment:
I wasn't 100% sure on doing this. I provide the raw time stamp and the `CoscradDate` to the client, the latter as a view concern.

Reply:
I had to sort out an issue around `Date` methods being dependent on environment. Going forward, everything needs to be `UTC` and we need to use `UTC` specific methods. I noticed this when snapshots were failing on the CI but not on my machine due to a time zone difference. Luckily the dummy data I used was less than 7 hours from midnight.